spec/models/aggregated_cash_income_spec.rb

```diff
@@ -85,12 +85,12 @@
     end
 
     context "when cash income transaction records exist" do
-      let!(:pension1) { create(:cash_transaction, :credit_month1, legal_aid_application: application, transaction_type: pension) }
-      let!(:pension2) { create(:cash_transaction, :credit_month2, legal_aid_application: application, transaction_type: pension) }
-      let!(:pension3) { create(:cash_transaction, :credit_month3, legal_aid_application: application, transaction_type: pension) }
-      let!(:maintenance_in1) { create(:cash_transaction, :credit_month1, legal_aid_application: application, transaction_type: maintenance_in) }
-      let!(:maintenance_in2) { create(:cash_transaction, :credit_month2, legal_aid_application: application, transaction_type: maintenance_in) }
-      let!(:maintenance_in3) { create(:cash_transaction, :credit_month3, legal_aid_application: application, transaction_type: maintenance_in) }
+      let!(:pension_first_month) { create(:cash_transaction, :credit_month1, legal_aid_application: application, transaction_type: pension) }
```

Comment:
You actually renamed them all 👏 🏆

Reply:
Only the ones where the numbers are greater than 2! :)
There are still a lot of variable1, variable2 pairs!

Reply:
There are still 82 offences, so 41 occurrences of numbered variables
Happy to discuss if the team are happy with more 1&2 being created but 1,2&3 will get blocked?